mne/channels/montage.py

```diff
@@ -929,6 +974,9 @@ def read_dig_montage(hsp=None, hpi=None, elp=None,
         )
 
     elif bvct is not None:
+        warn('Using "read_dig_montage" with "bvct" not None'
+             ' is deprecated and will be removed in v0.20.'
+             ' Please use read_dig_captrack instead.', DeprecationWarning)
```

Comment:
this is message is also in _read_dig_montage_deprecation_warning_helper ?
is this on purpose?

Reply:
nop I just forgot to remove it.